patch/attiny84.yaml

```diff
@@ -0,0 +1,221 @@
+        ADC7_ADC7_1X: [38, "Differential Inputs Positive ADC7 Negative ADC7 1x Gain"]
+        ADC7_ADC7_20X: [39, "Differential Inputs Positive ADC7 Negative ADC7 20x Gain"]
+    REFS:
+      _write_constraint: enum
+      _replace_enum:
+        VCC: [0, "Vcc used as Voltage Reference, disconnected from Aref"]
+        AREF: [1, "External Voltage Reference at AREF pin, Internal Voltage Reference turned off"]
+        INTERNAL: [2, "Internal 1.1V Voltage Reference"]
+CPU:
+  _modify:
+    OSCCAL:
+      access: read-write
+  CLKPR:
+    CLKPS:
+      _replace_enum:
+        PRESCALER_1: [0, "Prescaler Value 1"]
+        PRESCALER_2: [1, "Prescaler Value 2"]
+        PRESCALER_4: [2, "Prescaler Value 4"]
+        PRESCALER_8: [3, "Prescaler Value 8"]
+        PRESCALER_16: [4, "Prescaler Value 16"]
+        PRESCALER_32: [5, "Prescaler Value 32"]
+        PRESCALER_64: [6, "Prescaler Value 64"]
+        PRESCALER_128: [7, "Prescaler Value 128"]
+        PRESCALER_256: [8, "Prescaler Value 256"]
+  MCUCR:
+    _add:
+      BODS:
+        description: "BOD Sleep (available on some devices)"
+        bitOffset: 7
+        bitWidth: 1
+        access: read-write
+      BODSE:
+        description: "BOD Sleep Enable (available on some devices)"
+        bitOffset: 2
+        bitWidth: 1
+        access: read-write
+  OSCCAL:
+    _modify:
+      CAL:
+        access: read-write
+EEPROM:
+  EECR:
+    EEPM:
+      _replace_enum:
+        ATOMIC: [0, "Atomic (erase and write in one operation)"]
+        ERASE: [1, "Erase only"]
+        WRITE: [2, "Write only"]
+EXINT:
+  _modify:
+    GIFR:
+      access: read-write
+  MCUCR:
+    _delete:
+      ISC??:
+    _add:
+      ISC0:
+        description: "Interrupt Sense Control 0 bits"
+        bitOffset: 0
+        bitWidth: 2
+        access: read-write
+    ISC0:
+      _replace_enum:
+        LOW: [0, "The low level of INTx generates an interrupt request"]
+        TOGGLE: [1, "Any logical change on INTx generates an interrupt request"]
+        FALLING: [2, "The falling edge of INTx generates an interrupt request"]
+        RISING: [3, "The rising edge of INTx generates an interrupt request"]
+USI:
+  _modify:
+    USISR:
+      access: read-write
+  USICR:
+    _modify:
+      USICLK:
+        access: write-only
+      USITC:
+        access: write-only
+    USIWM:
+      _replace_enum:
+        DISABLED:        [0, "All detectors disabled. Port pins operates as normal."]
+        THREE_WIRE:      [1, "Three-wire mode. Uses DO, DI, and USCK pins."]
+        TWO_WIRE_SLAVE:  [2, "Two-wire mode (Slave). Uses SDA (DI) and SCL (USCK) pins."]
+        TWO_WIRE_MASTER: [3, "Two-wire mode (Master). Uses SDA and SCL pins."]
+    USICS:
+      NO_CLOCK: [0, "No Clock/Software clock strobe"]
+      TC0:      [1, "Timer/Counter0 Compare Match"]
+      EXT_POS:  [2, "External, positive edge"]
+      EXT_NEG:  [3, "External, negative edge"]
+  USISR:
+    _modify:
+      USIDC:
+        access: read-only
```

Comment:
Can we make the USI patch common for all the `ATtiny` chips or are the differences too big for that to be feasible?

Reply:
It's identical, I can make that change.

Reply:
Done, and I pulled out a couple more, too.